### PR TITLE
update go to 1.18

### PR DESCRIPTION
**Pull request by lennysgarage**
Signed-off-by: Jonathan Marcantonio <jmarcant@redhat.com>

https://github.com/stolostron/backlog/issues/23955

### CODE REVIEW

File: Makefile.prow [hunk 1]
```diff
@@ -57,12 +57,7 @@ lint:
 
 .PHONY: lint-all
 
-lint-all: lint-yaml lint-go
```
Comment (philipwu08) [inline Makefile.prow:60]: Sure we don't know to lint the yaml files?

Comment (lennysgarage) [reply Makefile.prow:60]: My mistake, we still do want to lint the yaml files. I was just having an issue with pip install on linux as the executables are not referenced unless I add to path like so: `export PATH="$HOME/.local/bin:$PATH"`

**Pull request merged**